src/styles/_catalog.scss

```diff
@@ -19,4 +17,16 @@
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
```

Comment:
You already have grid-gap, it is no need to set margins.

Reply:
Oh that`s right. I forgot about this one!